communications/newsletters/newsletter_33_Apr2022.md

```diff
@@ -0,0 +1,259 @@
+_Congratulations to the participants and presenters at #CW22!_
+
+- Achintya Rao and his team members developed a [Carpentries-style](https://carpentries.org/) lesson on R's bug tracking process, that was awarded third place during the hack day.
+- Malvika Sharan and her team members developed collaboration project idea, "Hi-Brid! Leaving no one behind in hybrid working!" that won second place during the Collaborative Idea's session.
+- Jez Cope, Stephan Druskat, and Anne Lee Steele were awarded third place for their idea of creating a "Interdisciplinary Glossary of Glossaries".
+
+Many Turing Way core team members also gave talks and workshops about various projects: including Sarah Gibson, Emma Karoune and Vicky Hellon.
+[Look at their slides on Figshare here](https://ssi-cw.figshare.com/Collaborations_Workshop_2022_CW22).
+
+Congrats to you all on a great Collaborations Workshop!
+Thank you to the organisers, and to Rachael Ainsworth for keeping us on schedule!
+
+### Interview with Malvika
+
+![Alt:Screenshot of online interview with Raniere and Malvika](https://i.imgur.com/frLHFvD.jpg)
+_Discussing the importance of open science and reproducibility in research._
+
+Malvika Sharan was recently interviewed by Raniere Silva for [Reproducible Research Scout](https://gitlab.com/reproducible-research-scout), a group dedicated to computational reproducibility and related topics.
+[Watch the recording here](https://www.youtube.com/watch?v=GMwRyAwnlEk)!
+
+### Community Research
+
+![Alt:Screenshot of miro board with virtual stickynotes](https://i.imgur.com/yxPzAxH.jpg)
+_Brainstorming the role of tools and research infrastructure within The Turing Way_
+
+Over the past few weeks, Anne has been doing community research within _The Turing Way_, learning from contributors to the project at all stages.
+She posts [weekly updates on Friday on this Github Issue](https://github.com/alan-turing-institute/the-turing-way/issues/2318).
+If you'd like to get involved in the process or talk to her more about it, reach out to her directly on Slack or by [email](mailto:asteele@turing.ac.uk).
+Alternatively, you can also [schedule time in her calendly](https://calendly.com/aleesteele/).
+
+### Research on Open Governance Models
+
+![Alt:Screenshot of Github pull request](https://i.imgur.com/F30aRgk.jpg)
+_Check out the Github issue to learn more about the research done so far on open governance models_
+
+Malvika and Anne are also co-leading an investigation into open research governance practices in the fifth cohort of [Open Life Science program](https://openlifesci.org/) (OLS-5).
+This research will inform the next stage of mature governance structure in _The Turing Way_.
+We would love to involve members of our community in this research as well as , please comment on [Pull Request #31 on OLS-5 Github](https://github.com/open-life-science/ols-5/issues/31) or reach out to us on Slack directly.
```

Comment:
"as well as...." phrase feels unfinished

Reply:
Thanks for catching that1

Reply:
```suggestion
We would love to involve members of our community in this research, please comment on [Pull Request #31 on OLS-5 Github](https://github.com/open-life-science/ols-5/issues/31) or reach out to us on Slack directly.
```

Again that was me! I did some editing and for sure got distracted.